docs/book/src/tasks/experimental-features/runtime-sdk/index.md

```diff
@@ -16,6 +16,12 @@ Please note Runtime SDK is an advanced feature. If implemented incorrectly, a fa
 
 </aside>
 
+<aside class="note warning">
+
+Please note that hooks are only invoked for Clusters created using ClusterClass.
```

Comment:
Can we merge this note with the one at L7?

Reply:
Okay. done. thx.